lms/static/sass/_pearsonx.scss

```diff
@@ -213,9 +213,16 @@ h2 {
         width: 100%;
         height: auto;
       }
+      // Hide the picture for small screens OC-4050
+      @media screen and (max-width: 760px) {
+        display: none;
+      }
     }
     .contact-form-container {
-      flex-basis: 40%;
+      // Don't restrict form width when on small screens with no contact picture
```

Comment:
@xitij2000 If we need to keep the rule below, let's mention the ticket number here as well:

```css
// OC-4050: Don't restrict form width when on small screens with no contact picture.
```

Reply:
@itsjeyd  I don't think we *need* to keep this change. It doesn't look too bad without it, and only affects a small range of screen sizes. I'll amend this commit with the updated comments and push. If this change still seems unnecessary I can remove it.

Reply:
@xitij2000 Sounds good re: updating the comments. I replied [above](https://github.com/open-craft/edx-theme/pull/35#discussion_r171263022) regarding the impact of the second rule.